tests/helpers.py

```diff
@@ -144,7 +178,8 @@ def path_to_executable(program_name):
     for executable in [
         os.path.join(path, program_name),
         os.path.join(path, program_name + ".EXE"),
-        os.path.join(path, program_name + ".exe")]:
+        os.path.join(path, program_name + ".exe"),
+        os.path.join(path, "Debug", program_name + ".exe"),]:
```

Comment:
This doesn't look like a SIG_STFL specific addition: Why is this necessary? Why wasn't it necessary before?

Reply:
I add this pattern. I realize that the CI on Windows failed because it could not find the execution path. Inserting the `Debug` into the path make this work. 
I think it's necessary, whether we enable the debug flag or, this change makes the CI happy.

Reply:
Hmm -- what if we build a "Release" variant? Wouldn't the "Debug" have to be replaced by "Release" then? But then again, this is for Windows that I don't really know (or we all together care about :). So thanks for the explanation. Closing as Windows is not a fully supported platform.